docs/src/main/asciidoc/_high-availability/subsystem-support/Distributable_Web_Applications.adoc

```diff
@@ -96,6 +96,43 @@ e.g. Creating a new session management profile, using ATTRIBUTE granularity with
 }
 ----
 
+The subsystem also supports session management where session data is stored in a remote infinispan-server cluster via the HotRod protocol.
+
+e.g.
+
+[source]
+----
+[standalone@embedded /] /subsystem=distributable-web/hotrod-session-management=foo:add(remote-cache-container=datagrid, granularity=ATTRIBUTE)
+{
+    "outcome" => "success"
+}
+----
+
+remote-cache-container::
+This references a remote-cache-container defined in the Infinispan subsystem into which session data will be stored.
```

Comment:
@pferraro as a suggestion I would also add a sample cli statement for creating the cache in the Infinispan subsystem

Reply:
Examples like this belong in the Infinispan subsystem documentation - but we can link that from here.